Mathlib/MeasureTheory/Integral/CircleIntegral.lean

```diff
@@ -210,6 +212,19 @@ protected theorem finsum {ι : Type*} {f : ι → ℂ → E} (h : ∀ i, CircleI
 nonrec theorem neg (hf : CircleIntegrable f c R) : CircleIntegrable (-f) c R :=
   hf.neg
 
+/--
+If `f` is circle integrable, then so are its scalar multiples.
```

Comment:
This could also go on one line. (This might be my personal opinion only.) Same for the other doc-string